portal/src/app/hooks/useDashboardData.ts

```diff
@@ -10,13 +10,22 @@ export interface IUseDashboardData {
   testSuiteId: string;
   status: FetchDataStatus;
   handleRunQueryClick: (queryData: ITestParams) => void;
-}
+};
+
+interface ITestRequestData {
+  experimentName: string;
+  algorithms: string[];
+  iterationsCount: number[];
+  messageSizes: number[];
+  description: string;
```

Comment:
required or optional ?

Reply:
The description is not required, but with default value of empty string.
So it's fine to keep it that way.